stubs/simplejson/simplejson/__init__.pyi

```diff
@@ -32,17 +32,42 @@ def dumps(
     ignore_nan: bool = ...,
     int_as_string_bitcount: int | None = ...,
     iterable_as_array: bool = ...,
+    *,
```

Comment:
Thanks for adding the overloads!

I think we need to make everything from `indent` onwards keyword-only, because otherwise we'd incorrectly type calls with 6+ positional arguments.